src/Components/MainPage/TrendList/trendlist.css

```diff
@@ -0,0 +1,164 @@
+.trending-container {
+  max-width: 1250px;
+  width: 80vw;
+  margin: 0 auto;
+}
+
+.trending-title-wrapper {
+  display: flex;
+  text-transform: uppercase;
+  align-items: center;
+}
+
+.trending-title {
+  margin-right: 30px;
+  color: #493871;
+}
+
+.btn {
+  font-family: "Oswald", sans-serif;
+  background-color: transparent;
+  color: #493871;
+  border: 2px solid #493871;
+  padding: 9px;
+  letter-spacing: 5px;
+  border-radius: 10px;
+  outline: 0;
+  cursor: pointer;
+  font-weight: bold;
+  margin-right: 10px;
+  margin-left: 10px;
+  transition: 0.5s;
+  height: 45px;
+  width: 100px;
+}
+
+.btn:hover {
+  box-shadow: 2px 0px 63px -20px rgba(73, 56, 113, 1);
+}
+
+.active {
+  background-color: #ff7955;
+  border: 1px solid #ff7955;
+  color: #fff;
+}
+
+.trending-movie-container {
+  display: flex;
+  flex-wrap: no-wrap;
+  overflow: auto;
+  margin-bottom: 75px;
+}
+
+.trending-movie-container::-webkit-scrollbar {
+  height: 10px;
+  background-color: rgba(255, 121, 85, 0.1);
+  border-radius: 9px;
+}
+
+.trending-movie-container::-webkit-scrollbar-thumb {
+  background-color: #ff7955;
+  border-radius: 9px;
+  background-clip: content-box;
+  width: 5px;
+}
+
+.trending-link {
+  text-decoration: none;
+  color: #493871;
+}
+
+/*TrendingMovie*/
+
+.trending-movie {
+  position: relative;
+  margin: 10px;
+  max-width: 190px;
+  width: 100%;
+  flex: 0 0 auto;
+}
+
+.avg-vote {
+  position: absolute;
+  margin-left: 155px;
+  margin-top: -5px;
+  background-color: #f4327f;
+  color: white;
+  padding: 9px;
+  border-radius: 8px;
+  font-size: 12px;
+  width: 17px;
+  text-align: center;
+  z-index: 1;
+}
+
+.trending-movie-poster-wrapper {
+  border-radius: 5px;
+  height: 268px;
+  width: 184px;
+  display: inline-block;
+  overflow: hidden;
+  border: 2px solid#FF7955;
+}
+
+.trending-movie-poster {
+  transition: 0.4s;
+  height: 100%;
+  width: 100%;
```

Comment:
For height: auto I understand how it is better to avoid stretching image, but how would you then fix a situation like this where it doesn't fill up the entire div?
Would adding a min-width: 100% be an okay solution or no since it is still stretched then?


![Capture](https://user-images.githubusercontent.com/56635645/95016355-2b1c5600-065b-11eb-897d-1fd36d0e0abc.PNG)